content/applications/services/helpdesk/advanced/close_tickets.rst

```diff
@@ -1,48 +1,105 @@
+===============================
+
+As work on a ticket progresses, it is moved along to the next stage in the pipeline. Once the issue
+is solved, the ticket is moved to a *folded* stage, to signify the work has been completed. This
+marks the ticket as *closed*.
+
+To fold a stage, navigate to the :guilabel:`Helpdesk` dashboard and click on a team to open the
+pipeline. Choose a stage, and click on the gear icon at the top right of that stage's kanban column.
+
+.. image:: close_tickets/closing-edit-stage-gear.png
+   :align: center
+   :alt: View of stage on Helpdesk pipeline with emphasis on gear icon and edit stage option.
+
+.. warning::
+   Clicking the gear icon also displays the option to :guilabel:`Fold` the stage. This setting folds
+   the stage **temporarily** to simplify the kanban view. This does **not** close the tickets in
+   this stage. It also does not permanently fold the stage. If a stage needs to be folded so the
+   tickets can be marked as closed, continue following the steps below.
+
+Select :guilabel:`Edit Stage`. This will open the stage's settings page. Check the box labeled
+:guilabel:`Folded in Kanban`. :guilabel:`Save & Close`.
+
+   .. image:: close_tickets/closing-folded-setting.png
+      :align: center
+      :alt: Stage settings page.
+
+Automatically closing inactive tickets
 ======================================
 
-Allowing customers to close their own tickets gives them autonomy and minimizes misunderstandings
-around when an issue is considered solved or not. This results in operational capacity for support
-teams, and higher satisfaction for the customer.
+Tickets that are inactive for a set period of time can be automatically closed. At that point,
+they will be moved to a folded stage.
 
-Enable ticket closing
-=====================
+Go to the team's settings page by going to :menuselection:`Helpdesk --> Configuration --> Teams`.
+Under the :guilabel:`Self-Service` section, enable :guilabel:`Automatic Closing`.
 
-Start by navigating to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and choose
-the appropriate team. Then click :guilabel:`Edit` and enable :guilabel:`Ticket closing` by
-checking the field box.
+If one of the team's stages is set to be folded in the kanban view, it will be the default selection
+in the :guilabel:`Move to Stage` field. If the team has more than one folded stage, the stage that
+occurs first in the pipeline will be the default. If no stage is folded, the default selection will
+be the last stage in the pipeline.
 
-.. image:: close_tickets/close-ticket-settings.png
-   :alt: Ticket closing feature in Odoo Helpdesk.
+The :guilabel:`After days of inactivity` field defaults to `7`, but can be adjusted if necessary.
 
-To designate which stage the ticket migrates to once it is closed, navigate to the ticket pipeline
-by going to :menuselection:`Helpdesk --> Overview` and clicking :guilabel:`Tickets` on the team's
-card.
+.. warning::
```

Comment:
I would use these warning blocks sparingly as they appear as bad. Maybe use important blocks here as the user is not deleting their DB or causing something that can't be changed. 

How is the days of inactivity calculated in?